scripts/release/create_release_PR_commit.sh

```diff
@@ -0,0 +1,138 @@
+#!/bin/bash
+set -eu
+
+# Folder, where we have cloned repositories' sources
+SOURCES_DIR="sources"
+
+CHANGELOG_HEADER_LINES="6"
+
+FORK_DIR="GH_USERNAME-timescaledb"
```

Comment:
can we use git user.name here and use that so we dont have to do the sed thing in the other script

Reply:
maybe use `gh auth status` to get github user

Reply:
Alright, modified accordingly.